cypress/integration/WebInterface/Smoke Tests/EditMeasure.spec.ts

```diff
@@ -40,7 +51,9 @@ describe('Edit Measure', () => {
         cy.get(EditMeasurePage.measureStewardConfirmaionText).should('contain.text', 'Measure Steward Information Saved Successfully')
 
         //Navigate back to Measures page and verify if the Measure Name is updated
-        cy.get(Header.measures).click()
```

Comment:
why did we need to make this change? is clicking measures in the header not working?